src/aura/STG_CMP_Tabs/STG_CMP_Tabs.cmp

```diff
@@ -32,7 +35,8 @@
              <c:STG_CMP_Addr hierarchySettings="{!v.hierarchySettings}" isView="{!v.isView}" namespacePrefix="{!v.namespacePrefix}" />
          </div>
          <div aura:id="systemTabContent" class="site-code--content slds-scrollable--x slds-tabs__content" role="tabpanel">
-             <c:STG_CMP_System hierarchySettings="{!v.hierarchySettings}" isView="{!v.isView}" namespacePrefix="{!v.namespacePrefix}" />
+             <c:STG_CMP_System hierarchySettings="{!v.hierarchySettings}" accRecTypes="{!v.accRecTypes}" accRecTypeId="{!v.accRecTypeId}" 
+             accRecTypeName="{!v.accRecTypeName}" isView="{!v.isView}" namespacePrefix="{!v.namespacePrefix}" />
```

Comment:
help me understand why we need to carry the namespace prefix here - the component doesn't recognize it?